src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,32 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo | null,
```

Comment:
maybe don’t need to render this modal if you don’t have selected todo?
```suggestion
  todo: Todo,
```